sdc/io/csv_ext.py

```diff
@@ -388,9 +388,11 @@ def _sanitize_varname(varname):
 def to_varname(string):
     """Converts string to correct Python variable name.
     Replaces unavailable symbols with _ and insert _ if string starts with digit.
+    In case where many underscores in a row, replace by 1 underscore.
```

Comment:
```suggestion
    Multiple underscores replaced with single one.
```

Reply:
This solution also removes underscores at start and end of the string.